docs/specs/account-linking.md

```diff
@@ -388,14 +426,20 @@ The above configs defined the account linking behavior of two login id types:
 
 The default values of each login id types are different. Please see the below table.
 
-| Login ID Type | Defaults                                                                                              |
-| ------------- | ----------------------------------------------------------------------------------------------------- |
-| `email`       | <pre><code>user_profile:<br/>&nbsp;&nbsp;pointer: "/email"<br/>action: error</code></pre>             |
+| Login ID Type | Defaults                                                                                               |
+| ------------- | ------------------------------------------------------------------------------------------------------ |
+| `email`       | <pre><code>user_profile:<br/>&nbsp;&nbsp;pointer: "/email"<br/>action: error</code></pre>              |
 | `phone`       | <pre><code>user_profile:<br/>&nbsp;&nbsp;pointer: "/phone_number"<br/>action: error</code></pre>       |
 | `username`    | <pre><code>user_profile:<br/>&nbsp;&nbsp;pointer: "/preferred_username"<br/>action: error</code></pre> |
 
 Therefore when not specified, all attempts to link a new login id to existing users or idenitities will result in error.
 
+## Account Linking in Promote Flow
+
+Account linking can occur in promote flow, however, only `error` is allowed to be the action of account linking.
```

Comment:
why?

Reply:
In promote flow, there is an existing user already. And if we allow actions such as "login_and_link", it means we support login to an existing user during promote flow. But login to an existing user in promote flow is not allowed at the moment.

Reply:
I think we should allow it as promote flow UX-wise is equals to login or signup from end users' perspective.

If their behaviour is different, it will be very difficult for developers to use this function at all. 

If it is technically impossible to do so (e.g. not sure which login ID to use), there maybe some other design issues on our system.